aicoe/sesheta/utils.py

```diff
@@ -52,17 +93,23 @@
     "erikerlandson": "Erik Erlandson",
     "fridex": "Frido Pokorny",
     "giorgoskarantonis": "Giorgos Karantonis",
+    "gkrumbach07": "Gage Krumbach",
     "goern": "Christoph Goern",
-    "Gregory-Pereira": "Gregory Pereira",
+    "gregory-pereira": "Gregory Pereira",
+    "harshad16": "Harshad Reddy Nalla",
     "harshad16": "Harshad Reddy Nalla",
     "hemajv": "Hema Veeradhi",
     "humairak": "Humair Khan",
     "kpostoffice": "Kevin Postlethwait",
     "larsks": "Lars Kellogg-Stedman",
     "llunved": "Daniel Riek",
     "mayacostantini": "Maya Costantini",
+    "mayacostantini": "Maya Costantini",
+    "meile18": "Viliam Podhajecký",
+    "pacospace": "Francesco Murdaca",
```

Comment:
and a couple more dupes here:

```suggestion
    "meile18": "Viliam Podhajecký",
```